packages/inputs-minimal/src/radio_group/index.jsx

```diff
@@ -24,33 +25,42 @@ const RadioGroup = ({
   items,
   name,
   value,
+  error,
   Layout = RadioGroupHorizontalLayout,
 }) => {
   const handleChange = (event, val) => {
     onChange(event, val);
   };
 
   return (
-    <Layout>
-      {items.map((item) => (
-        <Radio
-          key={item.value}
-          onChange={handleChange}
-          name={name}
-          label={item.label}
-          value={item.value}
-          disabled={item.disabled}
-          checked={value === item.value}
-        />
-      ))}
-    </Layout>
+    <>
+      <Layout>
+        {items.map((item) => (
+          <Radio
+            key={item.value}
+            name={name}
+            label={item.label}
+            value={item.value}
+            onChange={handleChange}
+            disabled={item.disabled}
+            checked={value === item.value}
+          />
+        ))}
+      </Layout>
+
+      <Meta
```

Comment:
I would suggest to put `Meta` into `Layout`. Based on the naming Layout should decide on the whole layout of the component, not only the radios.

Also I think it might be beneficial to have them all under one root element to help with positioning (`TextField` also has a root div)

Reply:
@iamuchejude no idea why you resolved the conversation because you neither responded nor did the suggested change

Reply:
I wrapped the components in a root element as recommended. Placing  the `<Meta>` in `<Layout>` because the layout will align meta with the radios. For example `<HorizontalLayout>` will align all including meta horizontally.

Reply:
The thing is, `RadioGroup` had no `className` prop because the `Layout` prop already takes care of all the placing. There even is already a usage of it.

https://github.com/good-hood-gmbh/business-frontend/blob/b3278f945479fe03e140353bd52beca56850723e/client/containers/poi_profile/components/poi_call_to_action/components/action_form/action_type.jsx#L7

With 'taking it into the Layout' I meant something like a render prop.

`<Layout meta={<Meta />}>radio1 radio2 radio3</Layout>`

Now it's a bit weird because you can pass in a Layout component but still can't control the root element or where the errors are placed. It's actually not the component layout anymore but only `ButtonsLayout`.

Reply:
Yeah, passing as a render prop makes sense. I'm just gonna create a new PR with the updates.